packages/flutter_tools/test/general.shard/compile_batch_test.dart

```diff
@@ -66,30 +66,6 @@ void main() {
     Platform: kNoColorTerminalPlatform,
   });
 
-  testUsingContext('passes no-gen-bytecode to kernel compiler in aot/release mode', () async {
-    when(mockFrontendServer.stdout)
-      .thenAnswer((Invocation invocation) => Stream<List<int>>.fromFuture(
-        Future<List<int>>.value(utf8.encode(
-          'result abc\nline1\nline2\nabc\nabc /path/to/main.dart.dill 0'
-        ))
-      ));
-    final KernelCompiler kernelCompiler = await kernelCompilerFactory.create(null);
-    await kernelCompiler.compile(sdkRoot: '/path/to/sdkroot',
-      mainPath: '/path/to/main.dart',
-      buildMode: BuildMode.release,
```

Comment:
Can we keep the test for compiling in release mode? We can check that other flags (like --aot) are properly passed.

Reply:
Done

Reply:
Thanks! Even more lgtm!